src/fury/thirdparty/MurmurHash3.h

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
This is copied from murmur hash https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp, though  Austin Appleby didn;t claim for copyright, maybe we should not add license?

Reply:
Yes. Thanks for your inputs. I'll update all exceptions a few later.

Reply:
FYI, we skip some file headers in `pom` for license-maven-plugin config.

And LICENSE file added some files licenses